contrib/zpages/src/main/java/io/opencensus/contrib/zpages/ZPageHandlers.java

```diff
@@ -201,5 +203,13 @@ private static void stop() {
     }
   }
 
+  // Sets the maximum number of elements as Integer.MAX_VALUE to enable RunningSpanStore.
+  // This method will only execute once even if called multiple times.
+  private static void enableRunningSpanStore() {
+    if (!(isRunningSpanStoreInitialized.getAndSet(true))) {
```

Comment:
```java
private volatile boolean isRunningSpanStoreInitialized = false;

private static void enableRunningSpanStore() {
  if (!isRunningSpanStoreInitialized) {
    synchronize(this) {
      if (isRunningSpanStoreInitialized) {
        return;  // Already initialized, small race
      }
      Tracing.getExportComponent().getRunningSpanStore().setMaxNumberOfSpans(Integer.MAX_VALUE);
      isRunningSpanStoreInitialized = true;
    }
  }
}

```

Reply:
lgtm

Reply:
Updated. (Note the variables and locks are static.)